src/main/java/jewellery/inventory/controller/OrganizationSaleController.java

```diff
@@ -0,0 +1,58 @@
+package jewellery.inventory.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.validation.Valid;
+import java.util.List;
+import java.util.UUID;
+import jewellery.inventory.dto.request.SaleRequestDto;
+import jewellery.inventory.dto.response.OrganizationSaleResponseDto;
+import jewellery.inventory.dto.response.ProductReturnResponseDto;
+import jewellery.inventory.dto.response.ResourceReturnResponseDto;
+import jewellery.inventory.service.OrganizationSaleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/organizations")
```

Comment:
could include sales here and omit it in the methods
```suggestion
@RequestMapping("/organizations/sales")
```